packages/quick_actions/quick_actions_android/android/src/main/java/io/flutter/plugins/quickactions/QuickActions.java

```diff
@@ -30,7 +29,7 @@
 import java.util.concurrent.TimeUnit;
 
 final class QuickActions implements AndroidQuickActionsApi {
-  protected static final String EXTRA_ACTION = "some unique action key";
+  static final String EXTRA_ACTION = "some unique action key";
```

Comment:
This was flagged as using `protected` in a `final` class. I'm not actually sure how this was compiling before though; QuickActionsPlugin uses this, but isn't a subclass of QuickActions.